Assets/Scripts/FrameScoreDisplay.cs

```diff
@@ -1,97 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using System.Xml.Serialization;
-using UnityEngine;
-using UnityEngine.UI;
-
-public class FrameScoreDisplay : MonoBehaviour
-{
-
-	public Text TotalScore;
-	public Text Bowl1Score;
-	public Text Bowl2Score;
-	public Text Bowl3Score;
-	public Text Strike;
-	//set on the UI for the 10th frame
-	public bool TenthFrame;
-	private int previousScore;
-
-	private void Start()
-	{
-		ResetScore();
-	}
-
-	public void SetFrameScore(int? score)
-	{
-		if (score != null)
-		{
-			TotalScore.gameObject.SetActive(true);
-			TotalScore.text = score.ToString();
-		}
-	}
-
-	public void SetRollScore(List<int> rolls)
-	{
-		if (!Bowl1Score.IsActive() && rolls.Count == 1)
-		{
-			previousScore = -1; //need to skip the spare check on the first bowl of a frame
-			SetRollScoreText(rolls[0], Bowl1Score);
-		}
-		else if (!Bowl2Score.IsActive() && rolls.Count == 2)
-		{
-			SetRollScoreText(rolls[1], Bowl2Score);	
-		} else if (TenthFrame && !Bowl3Score.IsActive() && rolls.Count ==3)
-		{
-			SetRollScoreText(rolls[2], Bowl3Score);
-		}
-	}
-
-	private void SetRollScoreText(int score, Text text)
-	{
-		if (score == 0)
-		{
-			text.text = "-";
-		}
-		else if (score + previousScore == 10) //Spare
-		{
-			text.text = "/";
-		}
-		else if(score == 10) 				//Strike
-		{
-			if (TenthFrame)				
-			{
-				text.text = "X";
-			}
-			else
-			{
-				text = Strike;    //Strike on anything but last frame takes up whole roll box
-				text.text = "X";			
-			}	
-		}
-		else 								//Open Frame
-		{
-			text.text = score.ToString();	
-		}
-		text.gameObject.SetActive(true);
-		previousScore = score;
-	}
```

Comment:
There's quite a lot of logic for figuring out "how to display" a score. With this amount of logic you're approaching the point where breaking it out into it's own class will make life easier in the long term.

That way if you needed to display the score in any other ways, or locations, you'd have the logic to do it all available, and it would just be a 'view' problem.